cmd/skaffold/app/skaffold_test.go

```diff
@@ -83,3 +83,24 @@ func TestSkaffoldCmdline_MainUnknownCommand(t *testing.T) {
 		t.CheckError(true, err)
 	})
 }
+
+func TestMain_InvalidUsageExitCode(t *testing.T) {
+	testutil.Run(t, "unknown command", func(t *testutil.T) {
+		// --interactive=false removes the update check and survey prompt.
+		t.Override(&os.Args, []string{"skaffold", "unknown", "--interactive=false"})
+		err := Run(ioutil.Discard, ioutil.Discard)
+		t.CheckErrorAndExitCode(127, err)
+	})
```

Comment:
For sake of completeness, can you add a test case for
- nil error exit code should be 0
- an error other than what matches the exit code 1

Reply:
never mind, i see it is covered in the test above.